### PR TITLE
feature(@desktop/communities): add loading state when import community

### PR DESCRIPTION
**Pull request by saledjenic**
Toast message is added when user access an existing community using community's private key. Toast
message with message that importing community is in progress is displayed while community is being
imported and once it is imported toast is closed and new one, which will be closed in 4 seconds,
with message that community is imported is displayed.

Fixes: #2467

### CODE REVIEW

File: ui/shared/ToastMessage.qml [hunk 1]
```diff
@@ -5,34 +5,51 @@ import "../imports"
...
     y: parent.height - height - Style.current.bigPadding
 
+
+    //% "View on Etherscan"
+    readonly property string defaultLinkText: qsTrId("view-on-etherscan")
+
+    property url source: "../app/img/check-circle.svg"
+    property color iconColor: Style.current.primary
+    property bool iconRotates: false
+    property string title: "Transaction pending..."
```
Comment (pinebit) [inline ui/shared/ToastMessage.qml:27]: not translated

File: ui/shared/ToastMessage.qml [hunk 2]
```diff
@@ -5,34 +5,51 @@ import "../imports"
...
-        }, 4000);
+        }, dissapearInMs);
     }
     onClosed: {
         // Reset props
+        source = "../app/img/check-circle.svg"
         iconColor = Style.current.primary
         iconRotates = false
-        root.linkText = defaultLinkText
+        title = "Transaction pending..."
```
Comment (pinebit) [inline ui/shared/ToastMessage.qml:47]: not translated

**Comment by saledjenic** [comment]
@pinebit for "Transaction pending..." text property particularly I just moved it, it was there before my changes. I add translations for all untranslated properties we have in this branch so far in a separate commit.

**Review by jrainville — CHANGES_REQUESTED** [review]
Nice job.

I'm requesting changes, because I think that displaying the private key is not a good idea.

Also, I don't think it was necessary to run the translation scripts, though it's good to do from time to time, so it's no big deal. At least you put the translation changes in a different commit, though both your commits have the same name, so I got confused. You should probably rename your second commit something like "chore: run translation scrips"

**Comment by saledjenic** [comment]
@jrainville I fixed what you requested. There are two commits one with my changes and another with translation related changes. But for it I noticed that many properties doesn't have translations not only properties from changes I applied here, what about that? Do we have translations for them or I should add them only for en language?

Another thing popped up while I tested this now, and it's related most likely to status-go (but I am not sure if such check may be even applied) when you try to import a community even you set some fake pk, community is imported, but without name. I guess that status-go should return kind of error in such case if that check is possible.

**Review by jrainville — APPROVED** [review]

**Pull request merged**